README.md

```diff
@@ -306,31 +333,35 @@ pull-filter ignore "auth-token" # fix PIA reconnection auth error that may occur
 Put it in the docker-compose file, and make deluge use the vpn container network:
 
 ```yaml
-vpn:
+  vpn:
     container_name: vpn
     image: dperson/openvpn-client:latest
     cap_add:
       - net_admin # required to modify network interfaces
     restart: unless-stopped
     volumes:
-      - /dev/net/tun:/dev/net/tun # tun device
-      - ${HOME}/.vpn:/vpn # OpenVPN configuration
+      - /dev/net:/dev/net:z # tun device
+      - ${ROOT}/config/vpn:/vpn # OpenVPN configuration
+    security_opt:
+      - label:disable
```

Comment:
Is that security option a requirement?

Reply:
In the most recent openvpn-client docker-compose example file it says it is in a comment, so I'd assume so. I haven't tested without it though https://github.com/dperson/openvpn-client/blob/master/docker-compose.yml#L6